### DISCUSSION TITLE
It is not backing up all the schemas I have from my own database

### DISCUSSION THREAD

**Discussion by edu-eng** [prodrigestivill/docker-postgres-backup-local#103] [Q&A] [answered]
Is there a parameter that I can pass to back up all my schemas within a single database? 
Today the backup of the default schema that would be public is being triggered.

**Comment by prodrigestivill** [comment, marked as answer]
My undestanding by default, pgdump backups all schemas.

To pass additional parameters use POSTGRES_EXTRA_OPTS.
Read the [README.md](https://github.com/prodrigestivill/docker-postgres-backup-local/blob/master/README.md#environment-variables) file for more information.

Specifically [read](https://www.postgresql.org/docs/12/app-pgdump.html#PG-DUMP-OPTIONS) about `--schema=pattern` and `--exclude-schema=pattern`
